ci/tasks/scripts/docker-semver-tags

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e -u
+
+version=$(cat version/version)
+latest_version=$(cat latest-version/version)
+
+major_version=$(echo $version | cut -d. -f1)
+minor_version=$(echo $version | cut -d. -f1,2)
+
+latest_minor_version=$(echo $latest_version | cut -d. -f1,2)
+
+echo $version       >> tags/tags
+echo $minor_version >> tags/tags
+
+# if the version we're publishing is for the latest minor version, bump the
+# major and latest tags, too
+if [ "$minor_version" = "$latest_minor_version" ]; then
```

Comment:
Might be missing something but don't we want major & latest to always be bumped as well when publishing a new image ?

Reply:
Or we only apply those tags after sometime X manually when we feel the image has baked sufficiently ?

Reply:
It's a bit tricky. We don't want to bump `5` or `latest` if the current latest release is 5.2.x and we just shipped 5.1.1 from a release pipeline. That would effectively "downgrade" them to an older minor release. Does that make sense?

Reply:
doh, yes, that does